js_modules/dagit/packages/core/src/graph/SVGViewport.tsx

```diff
@@ -596,3 +597,46 @@ const ZoomSliderContainer = styled.div`
   padding-bottom: 0;
   background: rgba(245, 248, 250, 0.4);
 `;
+
+const WheelInstructionTooltip: React.FC = () => {
+  const [usedMeta, setUsedMeta] = React.useState(false);
+  const [wheeling, setWheeling] = React.useState(false);
+  const timeout = React.useRef<NodeJS.Timeout>();
+
+  React.useEffect(() => {
+    const listener = (e: WheelEvent) => {
+      clearTimeout(timeout.current);
+      if (e.metaKey || e.shiftKey || e.ctrlKey) {
+        setUsedMeta(true);
```

Comment:
It seems like `usedMeta` can never be set to `false` -- do you definitely need both `usedMeta` and `wheeling` state variables? Or could it just be a single `visible` state value?